games-server/netmaumau/netmaumau-0.10.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 2015 Julian Ospald <hasufell@posteo.de>, Heiko Schaefer <heiko@rangun.de>
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit autotools flag-o-matic eutils libtool vcs-snapshot
+
+DESCRIPTION="Server for the popular card game Mau Mau"
+HOMEPAGE="http://sourceforge.net/projects/netmaumau"
+SRC_URI="https://github.com/velnias75/NetMauMau/archive/V${PV}.tar.gz -> ${P}-server.tar.gz"
+
+LICENSE="LGPL-3"
+SLOT="0/0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc static-libs -cli-client -xinetd"
```

Comment:
first: Lets just remove the ```-``` in front of those USE flags. It has almost no effect anyway compared to no ```-```. It's only used if you do smart things with the profile.

second: what exactly does xinetd do? Do we get linkage or does it just install files? In case we just install files we usually install such services unconditionally (like systemd services) and tell the user that/how he can use them.